src/CosmosDB/CosmosDB/CosmosDB.csproj

```diff
@@ -6,7 +6,8 @@
 
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.3.0" />
+    <PackageReference Include="Azure.Security.KeyVault.Keys" Version="4.3.0-beta.4" />
```

Comment:
>="4.3.0-be [](http://example.com/codeflow?start=68&length=10)

Any specific reason to use this beta package?

Reply:
as discussed offline.  As recommended by PS team, usage has to be uniform across.